toolkit/tools/internal/retry/retry_test.go

```diff
@@ -0,0 +1,83 @@
+package retry
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// Delay vs. retry count for mult = 1.0, delay = .1 seconds
+var expectedValuesMillis1 = []int{0, 100, 200, 300, 400, 500}
+
+// Delay vs. retry count for exp = 2.0, delay = 0.1 seconds
+var expectedValuesMillis2 = []int{0, 100, 400, 900, 1600, 2500}
+
+func TestCalcDelay1(t *testing.T) {
```

Comment:
Could we make the test names more descriptive? It's not too clear what `1` is about here.